### ISSUE TITLE
Allow address ranges to be marked as 'no auto-allocate'

### ISSUE THREAD

**Issue by mzealey** [OpenNebula/one#4030]
**Description**
We are migrating a large cluster of servers over many different networks/vlans to ONE. We don't want sysadmins to accidentally create a new server on ONE with an IP address of an existing server during this time. It would be nice to be able to set up most vlans with IPv4 address ranges but have a tickbox to say 'do not auto-allocate addresses from this range' so that allocation can only happen by someone specifying the IP address when network interface is added to the VM creation.

We could potentially do this using reservations but would have to delete each reservation and then create the vm with that IP address which seems not very user-friendly.

## Progress Status
- [ ] Branch created 
- [ ] Code committed to development branch
- [ ] Testing - QA
- [ ] Documentation 
- [ ] Release notes - resolved issues, compatibility, known issues
- [ ] Code committed to upstream release/hotfix branches
- [ ] Documentation committed to upstream release/hotfix branches

**Comment by tinova** [comment]
Would any of the following cover your use case:

  * hold the IPs of the AR and release one before launching  a VM with an IP in that range
  * for regular VMs, set the AR ID to the other ARs (the ones that allow for auto-allocation) so no IP is acquired automatically from the 'no auto allocate' AR

we want to keep the code as simple as possible

**Comment by mzealey** [comment]
yeah we ended up having to write a script to create ip ranges and mark them all as held. it's still pretty annoying to have to do this though

**Comment by tinova** [comment]
ok, i'm adding this to the backlog, thanks for the feedback
